src/scripts/components/forms/addAuthorForm.js

```diff
@@ -17,6 +17,10 @@ const addAuthorForm = () => {
         <label for="title">Email</label>
         <input type="email" class="form-control" id="email" aria-describedby="Email" placeholder="Enter Email" required>
       </div>
+      <div class="form-check">
+        <input type="checkbox" class="form-check-input" id="favorite" ${obj.favorite ? 'checked' : ''}>
```

Comment:
Nice! We should also make sure that the other inputs have a default value if provided!